src/regions.py

```diff
@@ -22,65 +22,59 @@
 from db_action_response import DbActionResponse
 import score_table_models as stm
 from score_table_models import Region as rg
+import db_utils
 
 from pandas import DataFrame
 from sqlalchemy.dialects.postgresql import insert
 from sqlalchemy.inspection import inspect
 
-MIN_LONG = -180.0
-MAX_LONG = 180.0
-
-HTTP_GET = 'GET'
-HTTP_PUT = 'PUT'
-
 PARAM_FILTER_TYPE = 'filter_type'
 
 FILTER__NONE = 'none'
 FILTER__BY_REGION_NAME = 'by_name'
 FILTER__BY_REGION_DATA = 'by_data'
 
-VALID_METHODS = [HTTP_GET, HTTP_PUT]
 VALID_FILTER_TYPES = [
     FILTER__NONE, FILTER__BY_REGION_NAME, FILTER__BY_REGION_DATA]
 
-EQUATORIAL = {'name': 'equatorial', 'min_lat': -5.0, 'max_lat': 5.0}
-GLOBAL = {'name': 'global', 'min_lat': -90.0, 'max_lat': 90.0}
-NORTH_HEMIS = {'name': 'north_hemis', 'min_lat': 20.0, 'max_lat': 60.0}
-TROPICS = {'name': 'tropics', 'min_lat': -20.0, 'max_lat': 20.0}
-SOUTH_HEMIS = {'name': 'south_hemis', 'min_lat': -60.0, 'max_lat': -20.0}
-TEST_SOUTH_HEMIS = {'name': 'test_south_hemis', 'min_lat': -50.0, 'max_lat': -35.0}
-
-DEFAULT_REGION_DEFS = [EQUATORIAL, GLOBAL, NORTH_HEMIS, TROPICS, SOUTH_HEMIS]
+EQUATORIAL = {'name': 'equatorial', 'min_lat': -5.0, 'max_lat': 5.0, 'min_lon': 0.0, 'max_lon': 360.0}
+GLOBAL = {'name': 'global', 'min_lat': -90.0, 'max_lat': 90.0, 'min_lon': 0.0, 'max_lon': 360.0}
+NORTH_MIDLAT = {'name': 'north_midlatitudes', 'min_lat': 20.0, 'max_lat': 60.0, 'min_lon': 0.0, 'max_lon': 360.0}
+TROPICS = {'name': 'tropics', 'min_lat': -20.0, 'max_lat': 20.0, 'min_lon': 0.0, 'max_lon': 360.0}
+SOUTH_MIDLAT = {'name': 'south_midlatitudes', 'min_lat': -60.0, 'max_lat': -20.0, 'min_lon': 0.0, 'max_lon': 360.0}
+TEST_SOUTH_HEMIS = {'name': 'test_south_hemis', 'min_lat': -50.0, 'max_lat': -35.0, 'min_lon': 0.0, 'max_lon': 360.0}
 
```

Comment:
This is not within the scope of the PR or the code base itself. This will be a user input decision made via running the code, not within the code base.